ruby-challenges.rb

```diff
@@ -22,13 +62,49 @@
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+          beatles_album1.delete 'aeiou'
+          beatles_album2.delete 'aeiou'
+          beatles_album3.delete 'aeiou' 
+            
+            puts beatles_album1, beatles_album2, beatles_album3
```

Comment:
Just remember that most Ruby built-in methods are accessors, which means this is not actually modifying the original string.